src/components/yeti/SubPanelCourse.vue

```diff
@@ -0,0 +1,308 @@
+<template>
+  <div class="yeti-subpanel panelCourse">
+    <sub-panel-title>Route</sub-panel-title>
+    <div v-if="features.length">
+      <div class="columns is-mobile">
+        <div class="column">
+          <p class="features-title">
+            <icon-route class="icon document-icon" />
+            <fa-icon icon="pen" class="icon edit-icon" />
+            <span
+              v-show="!hasFeaturesTitle"
+              class="features-title-text features-title-placeholder"
+              @click="onEditNewFeaturesTitle"
+              v-translate
+            >
+              Add title…
+            </span>
+            <!-- eslint-disable -->
+            <!-- allow to write {{...}} on same line to prevent too much spaces in contenteditable element -->
```

Comment:
The linter want this code to be something like

```js
<span
   ...
   >
   {{ ... }}
</span>
```

But this would lead problems, because it's a `contenteditable` element, so line return will be interpreted as spaces. We do want the source code to be `<span>{{...}}</span>` without spaces

Reply:
Prettier will handle this without failing (try it). The result will be:

```ts
          <span
              v-show="hasFeaturesTitle"
              class="features-title-text features-title-content"
              ref="featuresTitle"
              tabindex="0"
              contenteditable
              spellcheck="false"
              @blur="onEditFeaturesTitle"
              @keypress.13.prevent
              >{{ editableFeaturesTitle }}</span
            >
```
 As for eslint, there is no formatting rule included, so it won't say anything.

Reply:
OK, don't remember why I couldn't make it working then